test/iron-overlay-behavior.html

```diff
@@ -133,14 +141,23 @@
           assert.isFalse(spy.called, 'overlay should not refit');
         });
 
-        test('open() triggers iron-resize', function(done) {
+        test('open() & close() trigger iron-resize', function(done) {
           // Ignore iron-resize triggered by window resize.
           var callCount = 0;
           window.addEventListener('resize', function() { callCount--; }, true);
           overlay.addEventListener('iron-resize', function() { callCount++; });
-          runAfterOpen(overlay, function() {
-            assert.equal(callCount, 1, 'iron-resize should be called once');
-            done();
+          overlay.open();
+          listenOnce(overlay, 'iron-overlay-opened', function() {
```

Comment:
That would make the `close() triggers` test a replica of `open() triggers` test + another close. That's why I ended up merging these two.
But probably is better to separate these 2. I'll do it